regression/ansi-c/invalid_redeclaration1/test-same_type.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+-Dsame_return_type
+^EXIT=0$
+^SIGNAL=0$
+--
+Invariant check failed
```

Comment:
How do you check that is falling on the intended invariant violation here?

Reply:
This is a statement in the second of patterns not to occur, so this is checking that there is no invariant violation. I have added a comment to the file to explain which previous invariant violation this was.